src/operator/roi_pooling.cc

```diff
@@ -209,13 +207,21 @@ namespace mxnet {
 namespace op {
 
 template<>
-Operator* CreateOp<cpu>(ROIPoolingParam param) {
-  return new ROIPoolingOp<cpu>(param);
+Operator *CreateOp<cpu>(ROIPoolingParam param, int dtype) {
+  Operator* op = NULL;
+  MSHADOW_REAL_TYPE_SWITCH(dtype, DType, {
+    op = new ROIPoolingOp<cpu, DType>(param);
+  });
+  return op;
 }
```

Comment:
Thanks for the review. I didn't quite follow the `;` after `}` advice. Would you please elaborate on that?